src/SystemCommands-RefactoringSupport/RBPullUpPreview.class.st

```diff
@@ -0,0 +1,90 @@
+"
+I am a preview of Pull up refactoring, I can select the superClass of pull up methos, and select the methods of class to pull up
+"
+Class {
+	#name : #RBPullUpPreview,
+	#superclass : #RBPushDownPreview,
```

Comment:
It would be better to use unified naming like pushUp and pushDown (to not mix push and pull words). There was discussion about difference on mailing list. But I don't remember conclusions.